frontend/packages/ux-editor/src/components/config/editModal/EditOptions/OptionTabs/EditTab/OptionListEditor/LibraryOptionsEditor/LibraryOptionsEditor.module.css

```diff
@@ -0,0 +1,33 @@
+.editOptionTabModal[open] {
+  --code-list-modal-min-width: min(80rem, 100%);
+  --code-list-modal-height: min(45rem, 100%);
+
+  min-width: var(--code-list-modal-min-width);
+  height: var(--code-list-modal-height);
+}
+
+.content {
+  height: 100%;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.label {
+  font-weight: 500;
```

Comment:
why is this 500?

Reply:
Skal endre den til Bold, samme som brukes i `LocalImage.tsx`